tools/build.sh

```diff
@@ -1,11 +1,28 @@
 #!/bin/bash
 
-# Note: Not sure if we use this. It's not straightforward to install on OSX
-# if ! which envsubst > /dev/null 2>&1; then
-#  echo "You don't seem to have envsubst yet. Go get it! Using sed as an " \
-#    "alternative approach is too slow... :("
-#  exit 1
-# fi
+# execute config file given as parameter; default to testnet
+CONFIG_FILE=testnet
+if [ "$1" != "" ]; then
+    CONFIG_FILE="$1"
+fi
+
+source config/$CONFIG_FILE.conf
```

Comment:
Add a check if config file exists. Or does `source` fail the script when the file does not exist?

Reply:
yep, done